### PR TITLE
Add link to replace autocomplete if address is unknown by google

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Form/DonationRequestType.php [hunk 1]
```diff
@@ -117,7 +117,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('cityName', TextType::class, [
                 'required' => false,
             ])
-            ->add('country', UnitedNationsCountryType::class)
+            ->add('country', UnitedNationsCountryType::class, [
+                'preferred_choices' => ['FR'],
```
Comment (HeahDude) [inline src/Form/DonationRequestType.php]:
```diff
- 'FR'
+ Address::FRANCE
```
?

File: templates/donation/informations.html.twig [hunk 2]
```diff
@@ -65,7 +65,12 @@
         <div>
             {{ form_errors(form.address) }}
 
-            <div class="form__row address-autocomplete"></div>
+            <div class="form__row">
+                <div class="address-autocomplete"></div>
+                <p class="text--smallest text--muted text--left visually-hidden" id="address-autocomplete-help-message">
```
Comment (HeahDude) [inline templates/donation/informations.html.twig]: If I force displaying this field client side, can I click on it (though bypassing the autocomplete)?

Comment (ottaviano) [reply templates/donation/informations.html.twig]: yes, you can do it, as you can disable JS and bypass the autocomplete, or inter unknown address and this message will appear automatically

Comment (HeahDude) [reply templates/donation/informations.html.twig]: Ok!

File: src/Form/DonationRequestType.php (hunk 1 above)
Comment (ottaviano) [reply src/Form/DonationRequestType.php]: fixed, thanks 👍

**Pull request merged**